src/models/hooks/objectiveTopic.js

```diff
@@ -1,6 +1,28 @@
 import { Op } from 'sequelize';
 import { AUTOMATIC_CREATION } from '../../constants';
 
+const autoPopulateOnAR = (sequelize, instance, options) => {
+  // eslint-disable-next-line no-prototype-builtins
+  if (instance.onAR === undefined
+    || instance.onAR === null) {
+    instance.set('onAR', false);
+    if (!options.fields.includes('onAR')) {
+      options.fields.push('onAR');
+    }
+  }
+};
```

Comment:
Should we also "unset" this flag if an objective is removed from a draft AR?

Right now, I am seeing a scenario where a new objective is added to an AR, the tables get updated with the "onAR" true, but then after removal, the true value persists. The removed objective shows in the RTR section.